app/models/manageiq/providers/vmware/infra_manager/inventory/collector.rb

```diff
@@ -163,7 +177,12 @@ def cis_connect(vim)
   def disconnect(vim)
     return if vim.nil?
 
+    # sessionManager.Logout and close the http connection
     vim.close
+
+    # Cleanup the certificate authority file if it exists
+    ca_file&.close
```

Comment:
NOTE: close before unlink ensures the inode isn't sticking around even after the dentry is removed